pkg/api/addons_test.go

```diff
@@ -3255,6 +3255,10 @@ func TestSetAddonsConfig(t *testing.T) {
 					Name:    common.CloudNodeManagerAddonName,
 					Enabled: to.BoolPtr(false),
 				},
+				{
+					Name:    common.KubeDNSAddonName,
```

Comment:
This was an error that wasn't caught because kube-dns wasn't actually being validated in this UT.